contributing/giving-constructive-criticism.rst

```diff
@@ -0,0 +1,154 @@
+Giving constructive criticism
+=============================
+
+:doc:`Reviewing issues and pull request </contributing/community/reviews>` is a great way to get started with
+contributing to the Symfony community. Anyone can do it! But before
+you give a comment, take a step back and think, is what you are about
+to say actually what you indent?
+
+Communicating over the internet with nothing but text can pose a
+big challenge, especially if you remember the Symfony community
+is world-wide and is composed of a wide variety of people with differing
+idea's and opinions.
+
+Not everyone speaks English, is able to use a keyboard, and some might
```

Comment:
three clauses here is confusing, maybe:

`Not everyone speaks English or is able to use a keyboard, and some might...`

Also maybe `Dyslexia or similar conditions that affect their writing` rather than picking out that one condition

Reply:
`Not everyone speaks English or is able to use a keyboard, some might
have Dyslexia or similar conditions that affect their writing.`?

Reply:
Yep, that could also be done as 2 sentences (`... keyboard. Some ...`)